index.bs

```diff
@@ -1526,39 +1525,40 @@ In order to report a {{Gamepad/mapping}} of {{GamepadMappingType/xr-standard}} t
   <thead>
     <tr>
       <th>Axes</th>
-      <th><code>xr-standard</code> Location</th>
+      <th><code>xr-standard</code> Binding</th>
       <th>Required</th>
     </tr>
   </thead>
   <tbody>
     <tr>
       <td>axes[0]</td>
-      <td>Primary Touchpad/Joystick X</td>
-      <td>Yes</td>
+      <td>Touchpad X</td>
+      <td>If thumbstick is not present</td>
```

Comment:
Side question, since primary axes were already required before: Are we requiring that all `xr-standard` gamepads have either a touchpad or thumbstick?  What about simpler "remote-style" controllers?

Reply:
My thoughts when initially laying out the mapping were that we wanted to ensure that if something was advertised as `xr-standard` developers could depend on it having a certain baseline level of functionality. I still think that's a reasonable goal, though it's maybe less important if paired with the profiles concept. And yes, that would mean that some input devices (Like the Daydream controller) wouldn't qualify for `xr-standard`.

If we wanted to loosen up on that restriction and simply say that `xr-standard` only means that the inputs are mapped to a predictable place, and does not imply which of them are present then we could do so. I think we might want to look at expanding the proposed generic profile set to be a bit more descriptive in that case, with variants like:

 - `button-controller` 
 - `touchpad-controller` // This is basically just the daydream controller and.... anything else?
 - `touchpad-button-controller`
 - `thumbstick-controller` // Never seen a thumbstick controller without a separate trigger
 - `thumbstick-button-controller`
 - `touchpad-thumbstick-controller` // Is this actually a thing anywhere?
 - `touchpad-thumbstick-button-controller`

Reply:
> If we wanted to loosen up on that restriction and simply say that xr-standard only means that the inputs are mapped to a predictable place, and does not imply which of them are present then we could do so.

This aligns with what I've been thinking here.  Before, we only had one concept, `xr-standard`, overloaded to serve two purposes:
* `xr-standard` defined which controller parts **map** to which `buttons`/`axes` IDs
* `xr-standard` promised a minimum **profile** of controller parts that developers could assume were present

We've now split these into two distinct `mapping` and `profile` concepts in the API.  It seems reasonable to embrace that, simplifying the meaning of the `xr-standard` mapping to a pure ID mapping, and allowing profiles to make more specific promises.

Reply:
Should we continue this thread as part of #738?

Reply:
I support this direction.  I'd kinda been feeling the same way as I looked over the actual spec text changes.